src/os/filestore/FileStore.cc

```diff
@@ -2667,31 +2667,33 @@ void FileStore::_do_transaction(
 
     case Transaction::OP_CLONERANGE:
       {
```

Comment:
test is added https://github.com/ceph/ceph/pull/7325/commits/5da53e47885d1b581364f52e045d5fd057a10c65